### ISSUE TITLE
[BUG] RuntimeError: memory access out of bounds when calling /scratch/<slug>/opengraph-image

### ISSUE THREAD

**Issue by mkst** [decompme/decomp.me#1135]
**Describe the bug**

Opengraph image does not load, following error is seen in the yarn logs:
```
GET https://decomp.me/api/scratch/CgHet
GET https://decomp.me/api/scratch/CgHet/compile
GET https://decomp.me/api/preset/17
RuntimeError: memory access out of bounds
    at wasm://wasm/005420d6:wasm-function[30]:0x3d9f3
    at new o7 (/var/www/decomp.me/repo/frontend/.next/server/app/scratch/[slug]/opengraph-image/route.js:147:58693)
    at new ie (/var/www/decomp.me/repo/frontend/.next/server/app/scratch/[slug]/opengraph-image/route.js:147:63111)
    at render (/var/www/decomp.me/repo/frontend/.next/server/app/scratch/[slug]/opengraph-image/route.js:147:67140)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Object.start (/var/www/decomp.me/repo/frontend/.next/server/app/scratch/[slug]/opengraph-image/route.js:147:67445)
```

**To Reproduce**
```sh
curl https://decomp.me/scratch/CgHet/opengraph-image
```

**Expected behavior**
No error.

**Comment by mkst** [comment]
Looks to be related to this https://github.com/vercel/next.js/issues/61047.

**Comment by ethteck** [comment]
They replied - looks like this is fixed in 14.1.3 and above, so I'll make sure #1119 is there
